### ISSUE TITLE
Zoom images in gallery view 

### ISSUE THREAD

**Issue by HoWilgh** [k3b/APhotoManager#50]
Zooming the thumbnails in gallery view with two finger gesture or at least a setting in the settings dialog is necessary.

**Comment by k3b** [comment]
What is the benefit to zoom the thumbnail in the gallery view? Currently you can tab on the thumbnail in the gallery view to open the imageview. The imageview can be zoomed.

**Comment by HoWilgh** [comment]
No, in my opinion it is no duplicate because I asked this to improve usability not for making the pictures look better. 
If you can zoom in gallery view you will find images much faster, especially if they are grouped e.g. by year. 
If it is possible you should test the Sony album app. https://play.google.com/store/apps/details?id=com.sonyericsson.album

BR

**Comment by k3b** [comment]
Hello HoWilgh.

Thanks for feedback.

Does gallery zoom mean that yo want to change the number of images in gallery view?

example: on my landsape tablet i currently have 4 * 3 visible thumbnails in gallery view.

* in gallery-zoom-out i would see 8*6 smaller images 
* in zoom-in-gallery i would see 2 x 3 bigger images

is this what you mean?

**Comment by HoWilgh** [comment]
Yes, it basically only changes the number of images shows at once. This helps massively finding images. Some time indicator such as month and year would help additionally. 
BR Wilfried
